source/interface/azure_iot_mqtt.h

```diff
@@ -0,0 +1,380 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file azure_iot_mqtt.h
+ *
+ * @brief Definition for the Azure IoT MQTT.
+ *
+ */
+
+#ifndef AZURE_IOT_MQTT_H
+#define AZURE_IOT_MQTT_H
+
+#include <stdint.h>
+#include <stddef.h>
+
+#if defined( __cplusplus ) || ( defined( __STDC_VERSION__ ) && ( __STDC_VERSION__ >= 199901L ) )
```

Comment:
should we move this into a az_iot_platform.h file?

Reply:
I removed the bool reference and converted them to uint8_t

---
In reply to: [598022528](https://github.com/Azure/azure-iot-middleware-freertos/pull/44#discussion_r598022528) [](ancestors = 598022528)